portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Leonard Zhang's Photo Gallery</title>
+    <link rel="stylesheet" href="gallery_style.css">
+    <script src="change_picture_script.js"></script>
+  </head>
+  <body>
+    <p id="return-link"><a href="index.html"><< Go back to the main portfolio page</a></p>
+    <div id="content">
+      <h1>Welcome to my Photo Gallery!</h1>
+      <h2>Let's first see some pictures of mine.</h2>
+      <div class="image-thumbnails">
+        <a href="images/Leonard-1.jpg"><img src="images/Leonard-1.jpg"/></a>
```

Comment:
It's good practice to add `alt` attributes to images that describe the picture for people using screen readers.

Reply:
I will try to add alt attributes to all the images.